### ISSUE TITLE
The accuracy of DVS dataset

### ISSUE THREAD

**Issue by donghyunli** [BICLab/Spike-Driven-Transformer#12]
Hello, I have a question about the accuracy of CIFAR10-dvs and DVS128gesture datasets. I've tried many different seeds with given configs (For CIFAR10-dvs, I used spikingjelly, not .pt files), but the accuracy is lower than your proposed results. The accuracy of CIFAR10-dvs and gesture is around 77% and under 99% respectively. 

So,  I just wonder if there are other configs or settings like augmentation when you got the proposed accuracy. 

Thank you.

**Comment by jkhu29** [comment]
Hi, thanks for your interest.

**Comment by donghyunli** [comment]
Thank you for the answer. I. re-check the learning rate and version of CUDA or PyTorch.

**Comment by junkangfang** [comment]
Hello,i have the same question,can you tell me what is your current learning rate and version of CUDA or PyTorch?
